microsoft/testsuites/mdatp/mdatp.py

```diff
@@ -0,0 +1,125 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import pathlib
+
+from assertpy import fail
+
+from lisa import Node, SkippedException, TestCaseMetadata, TestSuite, TestSuiteMetadata
+from lisa.operating_system import Windows
+from lisa.sut_orchestrator import AZURE
+from lisa.testsuite import simple_requirement
+
+# some constants from  check-mdatp.sh
+# if any mdatp install in /etc/opt is found
+EXIT_MDATP_AGENT_INSTALLED = 1
+# if mdatp az extension is installed
+EXIT_MDE_INSTALLED = 2
+# if any log dirs are found
+EXIT_MDATP_LOGS_FOUND = 4
+# if an installation log is found
+EXIT_MDATP_INSTALL_LOGS_FOUND = 8
+# if an onboarding blob is found
+EXIT_ONBOARD_INFO_FOUND = 16
+
+
+@TestSuiteMetadata(
+    area="mdatp",
+    category="functional",
+    description="""
+        Test to verify there are no pre-installed copies of mdatp.
+    """,
+)
+class MdatpSuite(TestSuite):
+    @TestCaseMetadata(
+        description="""
+           Check for mdatp endpoint/cloud install, dump config info.
+           Fails if mdatp is installed in the image.
+           Raises specific error messages depending on the type of info
+           foud
+        """,
+        priority=3,
+        requirement=simple_requirement(
+            unsupported_os=[Windows], supported_platform_type=[AZURE]
```

Comment:
how about BSD?

Reply:
It's supported.

Reply:
Not that there are any images in the marketplace for this, but MacOS should also be supported.